src/utils/twitch.ts

```diff
@@ -1,9 +1,11 @@
+const isDevelopment = process.env.NODE_ENV !== "production";
```

Comment:
use Enum instead of text comparation

```javascript

// On Header or another file
enum Environment {
Development = "development",
Production = "production",
}
// On line
const isDevelopment = process.env.NODE_ENV !== Environment.Production;
```

Reply:
That is just to avoid of string typo errors. 😉